cdk/lib/__snapshots__/index.test.ts.snap

```diff
@@ -417,27 +414,6 @@ exports[`The typerighter stack matches the snapshot 1`] = `
       },
       "Type": "AWS::EC2::SecurityGroupIngress",
     },
-    "DBSecurityGroupTyperighterrulemanagerfromtyperighterWazuhSecurityGroupF941BA2354320D59DB78": {
```

Comment:
Looks like these Wazuh changes may be slightly tricky to deploy: https://github.com/guardian/cdk/releases/tag/v61.5.0